### PR TITLE
Scoreboard

### PR DESCRIPTION
**Pull request by captureteam**
New component for displaying cool numbers, for example we use it in the analyze part.
image
image

### CODE REVIEW

File: src/mnd-bootstrap/components/_scoreboard.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_scoreboard.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/components/_scoreboard.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  </div>
+</div>
+```
+*/
+
+.scoreboard {
+  @include display(flex);
+  @include flex-direction(column);
+  @include align-items(center);
+  border-radius: $border-radius-base;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_scoreboard.scss]: Properties should be ordered background-color, border-radius, padding-bottom, padding-top

File: src/mnd-bootstrap/components/_scoreboard.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+.scoreboard-box {
+  @include flex(1);
+  text-align: center;
+}
+
+.scoreboard-box-number {
+  font-size: $font-size-very-large;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_scoreboard.scss]: Properties should be ordered color, font-size, line-height

File: src/mnd-bootstrap/components/_scoreboard.scss (hunk 1 above)
Comment (captureteam) [inline src/mnd-bootstrap/components/_scoreboard.scss]: Nope! It's the hologram documentation style

**Comment by captureteam** [comment]
:shipit:

**Pull request merged**